src/main/java/uk/gov/hmcts/reform/opal/repository/AntCtAmalgamatedRepo.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.hmcts.reform.opal.repository;
+
+//import org.springframework.data.jpa.repository.JpaRepository;
```

Comment:
Remove comments?

Reply:
Will be removed when these classes are updated for data access.